warning when running `grunt lint`

From https://github.com/mozilla/fxa-password-strength-checker/pull/12#discussion_r36041663

-----------------------

I see the following warning when running `grunt lint`:

```bash
➜  fxa-password-strength-checker git:(issue-3-verbose) grunt lint
Running "eslint:app" (eslint) task

src/bloomdata_short_pwd.js
  0:0  warning  File ignored because of your .eslintignore file. Use --no-ignore to override

✖ 1 problem (0 errors, 1 warning)
```

We generally treat warnings like errors - they shouldn't be there.

--- a/.eslintignore
+++ b/.eslintignore
@@ -1,4 +1,3 @@
 bower_components/**
 node_modules/**
-tests/node_modules/**
 src/bloomdata_short_pwd.js

--- a/tasks/eslint.js
+++ b/tasks/eslint.js
@@ -7,7 +7,7 @@ module.exports = function (grunt) {
   grunt.config('eslint', {
     app: {
       eslintrc: '.eslintrc',
-      src: ['Gruntfile.js', 'tasks/*.js', 'src/*.js']
+      src: ['Gruntfile.js', 'tasks/*.js', 'src/*.js', '!src/bloomdata_short_pwd.js']
     },
     test: {
       eslintrc: '.eslintrc',

